en/docs/learn/exporting-user-data-from-wso2-is.md

```diff
@@ -0,0 +1,28 @@
+#Exporting User Data from WSO2 Identity Server
+
+1. Download **scim-bulk-user-export-tool** from [here]().
+2. Extract the downloaded .zip file.
+3. Run the `start.sh` script inside the extracted directory.
+
+    `sh start.sh`
+
+4. First you will be prompted to provide the host address of the WSO2 Identity Server instance.
+
+    !!! Note
+        
+        If you’re getting user data from the super-tenant you only need to provide the host address of the instance 
```

Comment:
If you are getting data from the super-tenant,

Reply:
Fixed 9c3530b